src/components/dashboard/users/SectionTabs.tsx

```diff
@@ -10,28 +10,28 @@ const createLinkElement = (activeTab: string) => ({
         is="emby-linkbutton"
         data-role="button"
         class="${activeTab === 'useredit' ? 'ui-btn-active' : ''}"
```

Comment:
This file can probably be removed since tabs are no longer used in the dashboard navigation